dwave/preprocessing/include/dwave/presolve.hpp

```diff
@@ -148,6 +156,10 @@ class Presolver {
     const Postsolver<bias_type, index_type, assignment_type>& postsolver() const;
 
  private:
+    tf::Taskflow taskflowOneTime_;
```

Comment:
why break these into three rather than one big flow?

Reply:
That's mostly for a logical separation of the 3 different "types" of techniques. I'll do some more reading on the pros/cons and leave a note in the code either way.